### DISCUSSION TITLE
how to set promethus statefulset pod Args

### DISCUSSION THREAD

**Discussion by huangkaiyan10** [prometheus-operator/kube-prometheus#612] [General] [answered]
![image](https://user-images.githubusercontent.com/5146180/87768149-4b9ff580-c84e-11ea-9fb2-5033886d4560.png)

**Comment by huangkaiyan10** [comment]
how to set,at which yaml file?

**Comment by brancz** [comment, marked as answer]
You cannot. They are abstracted away in the `Prometheus` object.
